besca/export/_export.py

```diff
@@ -1132,12 +1133,20 @@ def pseudobulk(adata,
     data = adata.obs.get(column).to_frame(name=label)
     if data is None:
         sys.exit('please specify a column name that is present in adata.obs')
+        
+    data = adata.obs.get(column).to_frame(name=main_condition)
```

Comment:
I am confused. Should this be :

 data = adata.obs.get(main_condition).to_frame(name=main_condition)

Reply:
yes